src/main/scala/nl.knaw.dans.easy.deposit/EasyDepositApiApp.scala

```diff
@@ -210,12 +211,29 @@ class EasyDepositApiApp(configuration: Configuration) extends DebugEnhancedLoggi
    * @return
    */
   def getDepositState(user: String, id: UUID): Try[StateInfo] = {
+    trace(user, id)
+    getDepositStateManager(user, id)
+      .flatMap(_.getStateInfo)
+  }
+
+  def forceChangeState(user: String, id: UUID, newStateInfo: StateInfo, update: Boolean): Try[String] = {
+    trace(user, id)
+    for {
+      stateManager <- getDepositStateManager(user, id)
+      oldStateInfo <- stateManager.getStateInfo
```

Comment:
isn't this the same as `getDepositState` above?

Reply:
indeed, but now I don't have to create the `DepositDir` and `StateManager` again for the update.